content/docs/details/permissions/index.md

```diff
@@ -6,13 +6,15 @@ description: How to configure access control permissions in Roadie.
 
 ## Introduction
 
+**This feature is currently in the process of being rolled out. If you want access, please contact our support or sales teams.**
```

Comment:
Is this true? I believe this is enabling to everyone

Reply:
I changed it to the policies document, my mistake. Roles management has been rolled out, just not Policies management